e2e/helpers/auth/backends/Candid.ts

```diff
@@ -101,6 +99,9 @@ export class CandidUser extends LocalUser {
   }
 
   override async cliLogin() {
+    if (await juju.isUser(this.username)) {
```

Comment:
The login methods first checks if the user is already logged in and if so skips the login.